src/components/backupWallet/__tests__/__util__/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
```

Comment:
Why do we need this file duplicated all over

Reply:
@sirpy again, I just copied tests folder implementation from Auth.js

Reply:
I tried to move the content of the utils/index file to some common place from where the test files can fetch the required functions. But the implementation of this util/index requires to be at the same folder where the test is.